musicbot/bot.py

```diff
@@ -292,36 +292,29 @@ def _setup_logging(self):
             dlogger.addHandler(dhandler)
 
     @staticmethod
-    def _check_if_empty(
-        vchannel: discord.abc.GuildChannel, *, excluding_me=True, excluding_deaf=False
-    ):
-        def check(member):
-            if excluding_me and member == vchannel.guild.me:
-                return False
-
-            if excluding_deaf and any([member.deaf, member.self_deaf]):
-                return False
+    def _check_if_empty(vchannel: discord.abc.GuildChannel, *, deaf_as_inactive=True):
+        def is_inactive(member):
+            return (
+                member.bot
+                or (not member.voice)
+                or (deaf_as_inactive and (member.voice.deaf or member.voice.self_deaf))
+            )
 
-            if member.bot:
-                return False
+        return all(is_inactive(m) for m in vchannel.members)
```

Comment:
I love this return statement change but absolutely am not a fan of the refactor of the `_check_if_empty` function as I feel this is much less self documenting as how the function existed before. Would love a middle ground